pages/multi-token-collections/index.tsx

```diff
@@ -75,25 +96,71 @@ const fetchErc1155List = (variables: Variables): Promise<MultiTokenCollectionLis
     })
 
 const FILTER_KEYS = ['name']
+
 const MultiTokenCollectionList = () => {
   const [t] = useTranslation(['multi-token', 'common', 'list'])
   const {
-    query: { before = null, after = null, name = null, page_size = SIZES[1] },
+    push,
+    asPath,
+    query: {
+      before = null,
+      after = null,
+      name = null,
+      page_size = SIZES[1],
+      holder_count_sort,
+      name_sort,
+      type_count_sort,
+      ...restQuery
+    },
   } = useRouter()
 
   const title = t(`multi-token-collections`)
+
+  const handleSorterValues = (url, sorters) => {
+    const paramArr = url.slice(url.indexOf('?') + 1).split('&')
+    const sorterParams = []
+
+    paramArr.map(param => {
+      const [key, val] = param.split('=')
+      const decodeVal = decodeURIComponent(val)
+
+      sorters.includes(key) && sorterParams.push({ sort_type: decodeVal, sort_value: UdtsSorterValueEnum[key] })
+    })
+    return sorterParams
+  }
```

Comment:
A similar method was defined in https://github.com/Magickbase/godwoken-explorer-ui/pull/738/files#diff-dbc569892fd8b550b4a391217df328947c838a65ff30d015e970a47670e5280fR77